pkg/net/stdio/dial.go

```diff
@@ -0,0 +1,53 @@
+package stdio
+
+import (
+	"net"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+var execCommand = exec.Command
```

Comment:
Any reason for this indirection?

Apart from this, looks good to me, could you squash all the commits together, and add the PR description as the commit log?

Reply:
@cfergeau Using the variable was to make the tests here work.

https://github.com/sakai135/wsl-vpnkit/blob/v0.3.8/pkg/transport/dial_test.go